src/js/edu-benefits/containers/SubmitPage.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import SubmitMessage from '../components/SubmitMessage';
+
+class SubmitPage extends React.Component {
+  render() {
+		// TODO: shared address component?
+    const address = (
+      <address>
+        Western Region<br/>
+        VA Regional Office<br/>
+        P.O. Box 8888<br/>
+				Muskogee, OK 74402-8888
+      </address>
+		);
+
+		// TODO: replace this with actual data from the API
+    return (
+      <SubmitMessage
+          claimType="Education Benefits"
+          confirmation="8-910fks-76"
+          date="Jul 12, 2016"
+          address={address}
+          claimedBenefits="Chapter 33"
+          relinquishedBenefits="Chapter 1607"/>
```

Comment:
We could, although I'd rather see all of this information come back from the server.  If that proves to be tricky, we can definitely pull it from the store.